### ISSUE TITLE
utils.py-rescale_array()

### ISSUE THREAD

**Issue by gzc1090** [CVxTz/EEG_classification#6]
Hi! I have tested that rescale method affect largely on the results, so I am very curious  about why the rescale method  works.

**Comment by CVxTz** [comment]
Hello,

Neural networks are very sensitive to the scale of their input. Their loss function tend to be easier to optimize if the input to the neural network has a mean close to 0 and a std close to 1.
